src/rust/generators/osquery-subgraph-generator/src/generator.rs

```diff
@@ -20,89 +10,78 @@ use sqs_executor::{
         CompletedEvents,
         EventHandler,
     },
+    event_status::EventStatus,
 };
 
 use crate::{
     metrics::OSQuerySubgraphGeneratorMetrics,
-    parsers::PartiallyDeserializedOSQueryLog,
+    parsers::OSQueryEvent,
 };
 
 #[derive(Clone)]
-pub struct OSQuerySubgraphGenerator<C: Cache> {
+pub struct OSQuerySubgraphGenerator<C>
+where
+    C: Cache + Clone + Send + Sync + 'static,
```

Comment:
Cache is defined as 

```
pub trait Cache: Clone + Send + Sync
```

So you shouldn't need to add `+ Clone + Send + Sync` here as part of the trait bounds. I also assume that `'static` isn't needed here either.